### DISCUSSION TITLE
Editor Plug-In: Jump to the component definition

### DISCUSSION THREAD

**Discussion by gsalunke** [fastn-stack/fastn#608] [Ideas & RFCs]
Can we have a jump to definition feature in programming editors like VSCode?

Suggested by: @saurabh-lohiya

**Comment by gsalunke** [comment]
This can be done by implementing this:

[lsp-server for fastn #609](https://github.com/fastn-stack/fastn/discussions/609)

**Reply by saurabh-lohiya** [reply]
@gsalunke. Looking forward to this! Thank you

**Comment by amitu** [comment]
Closing in favour of #609.
